tests/Aspire.Dashboard.Tests/Integration/Playwright/PlaywrightFixture.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Workload.Tests;
+using Microsoft.Playwright;
+using Xunit;
+
+namespace Aspire.Dashboard.Tests.Integration.Playwright;
+
+public class PlaywrightFixture : IAsyncLifetime
+{
+    public IBrowser Browser { get; set; } = null!;
+
+    public async Task InitializeAsync()
+    {
+        PlaywrightProvider.DetectAndSetInstalledPlaywrightDependenciesPath();
+        Browser = await PlaywrightProvider.CreateBrowserAsync();
```

Comment:
This would create the browser instance, and launch the browser for every test class using this fixture. For workload tests we use:
https://github.com/adamint/aspire/blob/64ea4ab15976fbf903c9663aa6982205be8b1e2f/tests/Aspire.Workload.Tests/WorkloadTestsBase.cs#L22-L34
and `private static Lazy<IBrowser> Browser => new(CreateBrowser);`